templates/advantage/_table.html

```diff
@@ -0,0 +1,86 @@
+
+<div class="row">
+  <div class="table" id="enterprise-table">
+    <div class="row">
+      <div class="col-12 u-sv3" style="overflow-x: auto;">
+        <table class="p-table" style="width: 100%;">
+          <thead>
+            <tr>
+              <th><strong>Ubuntu Advantage for Infrastructure</strong></th>
```

Comment:
The copy doc says "What's included" here. I think what you've done makes more sense but not sure if it's intentional or not?

Reply:
Thanks, updated.